src/components/PaginationV2/PaginationV2.js

```diff
@@ -0,0 +1,228 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import Select from '../Select';
+import SelectItem from '../SelectItem';
+import { equals } from '../../tools/array';
+
+export default class PaginationV2 extends Component {
+  static propTypes = {
+    backwardText: PropTypes.string,
+    className: PropTypes.string,
+    itemRangeText: PropTypes.func,
+    forwardText: PropTypes.string,
+    id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    itemsPerPageText: PropTypes.string,
+    itemText: PropTypes.func,
+    onChange: PropTypes.func,
+    pageNumberText: PropTypes.string,
+    pageRangeText: PropTypes.func,
+    pageText: PropTypes.func,
+    pageSizes: PropTypes.arrayOf(PropTypes.number).isRequired,
+    totalItems: PropTypes.number,
+    disabled: PropTypes.bool,
+    page: PropTypes.number,
+    pageSize: PropTypes.number,
+    pagesUnknown: PropTypes.bool,
+    isLastPage: PropTypes.bool,
+    pageInputDisabled: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    backwardText: 'Backward',
+    itemRangeText: (min, max, total) => `${min}-${max} of ${total} items`,
+    forwardText: 'Forward',
+    itemsPerPageText: 'Items per page',
+    onChange: () => {},
```

Comment:
Instead of alloc'ing an empty function, could we just check for the `onChange` prop before calling it?